settings.gradle

```diff
@@ -18,7 +18,7 @@ plugins {
     // required to download the toolchain (jdk) from a remote repository
     // https://github.com/gradle/foojay-toolchains
     // https://docs.gradle.org/current/userguide/toolchains.html#sub:download_repositories
-    id("org.gradle.toolchains.foojay-resolver-convention") version "0.7.0"
+    id("org.gradle.toolchains.foojay-resolver-convention") version "1.0.0"
```

Comment:
- IBM_SEMERU was removed in Gradle 9.0.0
- Foojay plugin 0.9.0 still referenced this deprecated field
- Version 1.0.0 has removed these references and is Gradle 9 compatible